api/src/test/java/org/pmiops/workbench/interceptors/CronInterceptorTest.java

```diff
@@ -44,22 +44,22 @@ public void preHandleOptions_OPTIONS() throws Exception {
   @Test
   public void prehandleForCronNoHeader() throws Exception {
     when(request.getMethod()).thenReturn(HttpMethods.GET);
-    when(handler.getMethod()).thenReturn(OfflineAuditApi.class.getMethod("auditBigQuery"));
+    when(handler.getMethod()).thenReturn(OfflineUserApi.class.getMethod("synchronizeUserAccess"));
```

Comment:
It looks like this test used to use the outdated method, but now it uses OfflineUserApi. Does this mean that this is testing something else now?

Reply:
The CronInterceptor validates that API endpoints we've labeled as crons are invoked with a special App Engine cron header, which only App Engine can send to us:

https://cloud.google.com/appengine/docs/flexible/nodejs/scheduling-jobs-with-cron-yaml

While the prior method was deprecated, the test was still valid - the test just needs any cron method. `synchronizeUserAccess` is a different cron endpoint.